examples/server/server.cpp

```diff
@@ -3509,6 +3520,11 @@ int main(int argc, char **argv)
     }*/
     //);
 
+    if (sparams.n_threads_http > 0) {
+        log_data["n_threads_http"] =  std::to_string(sparams.n_threads_http);
+        svr.new_task_queue = [&sparams] { return new httplib::ThreadPool(sparams.n_threads_http); };
```

Comment:
Be default, I think number of http threads should be `n_parallel + 2` just to be sure, because the worst case:
- We have `n_parallel` requests running at the same time
- Plus one request for `/health`
- Plus one maybe for metrics endpoint

Reply:
P/s: These threads does almost nothing than just sitting and wait for `recv()`, so logically you can spawn how many you want without compromise performance.

In fact, modern async web framework doesn't even need thread pool, because it's done by using `epoll`. That should be ideal for use, but one major problem is that these framework are usually big, while what we want is just a single .hpp file

Reply:
Good point, but I prefer to keep the default httplib behavior for now and let the user decide if require. I would like to avoid people complaining to see lot of CPU threads created by the server because they are playing hard with GPU sequences and/or continuous batching.